shared/js/media/video_player.js

```diff
@@ -48,7 +53,7 @@ function VideoPlayer(container) {
   var durationText = newelt(slider, 'span', 'videoPlayerDurationText');
   // expose fullscreen button, so that client can manipulate it directly
   var fullscreenButton = newelt(slider, 'button',
-                          'videoPlayerFullscreenButton');
+                          'videoPlayerFullscreenButton', 'playbackFullscreen');
```

Comment:
I've never even seen the fullscreen button displayed. I think that is only used on tablets. I suppose it is worth fixing, but your patch may be untestable unless you have a tablet.